File: app/src/main/java/com/ciandt/dragonfly/example/config/CommonBundleNames.java
package com.ciandt.dragonfly.example.config;

import com.ciandt.dragonfly.example.BuildConfig;

/**
 * Created by iluz on 5/15/17.
 */

public interface CommonBundleNames {

    String BUNDLE_NAME_TEMPLATE = BuildConfig.APPLICATION_ID + ".%s";

    String SHOW_DEBUG_DRAWER_BUNDLE = String.format(BUNDLE_NAME_TEMPLATE, "SHOW_DEBUG_DRAWER_BUNDLE");

}

File: app/src/main/java/com/ciandt/dragonfly/example/config/CommonBundleNames.kt
package com.ciandt.dragonfly.example.config

import com.ciandt.dragonfly.example.BuildConfig

/**
 * Created by iluz on 5/15/17.
 */

interface CommonBundleNames {
    companion object {

        val BUNDLE_NAME_TEMPLATE = BuildConfig.APPLICATION_ID + ".%s"

        val SHOW_DEBUG_DRAWER_BUNDLE = String.format(BUNDLE_NAME_TEMPLATE, "SHOW_DEBUG_DRAWER_BUNDLE")

        val PROJECT_CHANGED = String.format(BUNDLE_NAME_TEMPLATE, "PROJECT_CHANGED")
    }
}
